app/Http/Controllers/Admin/Users/UserController.php

```diff
@@ -134,13 +137,33 @@ public function addUser()
     }
     
     /**
+     * Show a confirmation page to delete the selected user.
+     *
+     * @param int $id
+     *
+     * @return \Illuminate\View\View
+     */
+    public function delete(int $id)
+    {
+        $this->breadcrumbs->setCurrentRoute('admin.users.delete');
+        $user = $this->userRepository->find($id);
+        if (!$user) {
+            return redirect()->back()->withError(trans('errors.user_not_found'));
+        }
+        
+        return view('admin.users.delete', compact('user'))->withActive('users');
+    }
+    
+    /**
      * Delete the selected user
      *
+     * @param int $id
+     *
      * @return \Illuminate\View\View
      */
-    public function deleteUser(Request $request)
+    public function deleteUser(int $id)
```

Comment:
We are sending request over POST (because it prevent accidental user delete). You should undo your changes in this method.